### PR TITLE
Feature/8 input transparency

### PR DESCRIPTION
**Pull request by KudriavtsevAndrii**

### CODE REVIEW

File: src/Dali/RedSharp.Dali.Common.Interop/Native/Enums/WindowsLong.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace RedSharp.Dali.Common.Interop.Native.Enums
+{
+    internal enum WindowsLong : int
```
Comment (FlaviusHouk) [inline src/Dali/RedSharp.Dali.Common.Interop/Native/Enums/WindowsLong.cs:14]: Please leave a comment with some description of this enum.

File: src/Dali/RedSharp.Dali.Common.Interop/Native/Enums/WindowsStylesEx.cs [hunk 2]
```diff
@@ -0,0 +1,175 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace RedSharp.Dali.Common.Interop.Native.Enums
+{
+    [Flags]
+    internal enum WindowStylesEx : uint
```
Comment (FlaviusHouk) [inline src/Dali/RedSharp.Dali.Common.Interop/Native/Enums/WindowsStylesEx.cs:13]: Add some comments here please.

File: src/Dali/RedSharp.Dali.Common.Interop/Native/Enums/WindowsLong.cs (hunk 1 above)
Comment (KudriavtsevAndrii) [reply src/Dali/RedSharp.Dali.Common.Interop/Native/Enums/WindowsLong.cs:14]: Done.

File: src/Dali/RedSharp.Dali.Common.Interop/Native/Enums/WindowsStylesEx.cs (hunk 2 above)
Comment (KudriavtsevAndrii) [reply src/Dali/RedSharp.Dali.Common.Interop/Native/Enums/WindowsStylesEx.cs:13]: Done.

**Pull request merged**